content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -135,16 +182,19 @@ python scitt/create_hashed_signed_statement.py \
 1. View the Transparent Statement, as a result of registering the Signed Statement
 
     ```bash
-    python scitt/dump_cbor.py \
+    python datatrails_scitt_samples/dump_cbor.py \
       --input transparent-statement.cbor
     ```
 
+<!-- 
+TODO: Update with MMR verification
 1. Verify the signature of the receipt
 
     ```bash
-    python scitt/verify_receipt_signature.py \
+    python ${SCRIPTS}/verify_receipt_signature.py \
       --transparent-statement-file $TRANSPARENT_STATEMENT_FILE
     ```
```

Comment:
This does not verify the receipt signature. It verifies the transparency services counter signature on the signed statement.

It is impossible to verify a receipt signature without also verifying the includion proof (by design). This script should be called "verify_statement_counter_signature.py" or something